cmd/vm_resize.go

```diff
@@ -64,14 +63,13 @@ func resizeVirtualMachine(vmName string, diskValue int64) (*task, error) {
 
 	state := (string)(egoscale.VirtualMachineStopped)
 	if vm.State != state {
-		return nil, fmt.Errorf("this operation is not permitted if your VM is not stopped")
+		return nil, fmt.Errorf("this operation is not permitted if your Compute instance is not stopped")
```

Comment:
[nitpicking] Use positive/guidance form :)
```suggestion
		return nil, fmt.Errorf("this operation is not permitted while your Compute instance is running; stop it before issuing that command again")
```